carsus/io/output/base.py

```diff
@@ -106,6 +112,98 @@ def __init__(self,
 
         logger.info('Finished.')
 
+    @classmethod
+    def from_config(cls, fname):
+        with open(fname) as stream:
+            loaded_config = yaml.safe_load(stream)
+            loaded_config_keys = loaded_config.keys()
+
+            nist_config = loaded_config["nist"]
+            if nist_config["atomic_weights"] == "default":
+                atomic_weights = NISTWeightsComp()
+            else:
+                atomic_weights = NISTWeightsComp(
+                    nist_config["atomic_weights"]
+                )
+            ionization_energies = NISTIonizationEnergies(
+                nist_config["ionization_energies"]
+            )
+
+            gfall_config = loaded_config["gfall"] 
+            if gfall_config["path"] == "default":
+                # TODO: better alternative?
+                url = 'https://media.githubusercontent.com/media/tardis-sn/carsus-db/master/gfall/gfall_latest.dat'
+                os.system(f"wget -qO /tmp/gfall.dat {url}")
+                gfall_path = "/tmp/gfall.dat"
+            else:
+                gfall_path = gfall_config["path"]
+            gfall_reader = GFALLReader(
+                gfall_config["ions"],
+                gfall_path,
+                # TODO: priority
+            )
+
+            if loaded_config["zeta"]["path"] == "default":
+                zeta_data = KnoxLongZeta()
+            else:
+                zeta_data = loaded_config["zeta"]["path"]
+
+            if "chianti" in loaded_config_keys:
+                from carsus.io.chianti_ import ChiantiReader
```

Comment:
Maybe we can import this at the top of the file if `XUVTOP` env variable is in env.